index.html

```diff
@@ -394,13 +394,13 @@ <h3>Information for User Agents</h3>
 	      <li>Set the <abbr title="Document Object Model">DOM</abbr> focus:
 		<ol>
 		  <li>If the <a class="termref">element</a> can take <abbr title="Document Object Model">DOM</abbr> focus, the <a class="termref">user agent</a> MUST set the <abbr title="Document Object Model">DOM</abbr> focus to it.</li>
-		  <li>Otherwise, if the current element has an ID and an ancestor with the <pref>aria-activedescendant</pref> attribute present, and that ancestor is focusable, the user agent MUST set <abbr title="Document Object Model">DOM</abbr> focus to that ancestor.
+		  <li>Otherwise, if the current element has an ID and is <a>owned</a> by either a container element that has the <pref>aria-activedescendant</pref> property or by a container element that is controlled by an element that has the <pref>aria-activedescendant</pref> property, and the element that has <pref>aria-activedescendant</pref> is focusable, the user agent MUST set <abbr title="Document Object Model">DOM</abbr> focus to the element that has <pref>aria-activedescendant</pref>.
```

Comment:
Ditto to the first comment. This is now a substantive edit, no longer editorial.

If the WG agreed that substantive change is important, I'll concede that point, but I don't think the edit is clear as written either. For example: 

> …controlled by an element that has the aria-activedescendant property

The prose as written seems to apply to a controlling element that has an aria-activedescendant attribute with any value. Is that intentional?